packages/pytest-simcore/src/pytest_simcore/simcore_services.py

```diff
@@ -29,6 +29,7 @@
 
 
 _SERVICES_TO_SKIP: Final[set[str]] = {
+    "api-worker",

```

Comment:
minor: I hope this is not too much used in our fixtures or it might start failing.